Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Start with a base Ubuntu 22.04 image
```

Comment:
I suspect that we don't really want to add a Dockerfile at the top-level directory of the project.

Reply:
youre right this was mostly a testing dockerfile... ill move it :)